heclib/javaHeclib/src/Hec_zsqueeze.c

```diff
@@ -5,11 +5,8 @@
 JNIEXPORT jint JNICALL Java_hec_heclib_util_Heclib_Hec_1squeezeDSS
   (JNIEnv *env, jobject obj, jstring j_filename) {
 
-    char *filename;
-    int   status;
-
-    filename = (char *) (*env)->GetStringUTFChars (env, j_filename, 0); 
-	zsqueeze_ (filename, &status, strlen(filename));  
+   char* filename = (char *) (*env)->GetStringUTFChars (env, j_filename, 0); 
```

Comment:
```suggestion
   const char* filename = (const char *) (*env)->GetStringUTFChars (env, j_filename, 0); 
```

zsqueeze takes a `const char*`, I believe GetSTringUTFChars returns `const char*`, no reason to cast away the `const`